tools/stats/upload_test_stat_aggregates.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import ast
+import datetime
+import json
+import os
+import re
+from typing import Any, List, Union
+
+import rockset  # type: ignore[import]
+
+from tools.stats.upload_stats_lib import upload_to_s3
+
+
+def get_oncall_from_testfile(testfile: str) -> Union[List[str], None]:
+    path = f"test/{testfile}"
+    if not path.endswith(".py"):
+        path += ".py"
+    # get oncall on test file
+    try:
+        with open(path) as f:
+            for line in f:
```

Comment:
May be it's a good idea to also report test files that don't include the owners so that we know. Better yet is to have a lint check for this (optional) in a separate PR if you want to enforce this in the future